easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb

```diff
@@ -0,0 +1,24 @@
+name = 'MATLAB'
+version = '2018b'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# be sure to copy both DVD content to the SAME directory, 
+# including the hidden files, especially .dvd1 and .dvd2 
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8')]
+
+# Uncomment and modify the following variables if needed 
+# for installation with floating license server
+#license_server = 'my-license-server'
+#license_server_port = '1234'
+#key = '00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000'
+#modextravars = {'LM_LICENSE_FILE': '%s@%s' % (license_server_port, license_server)}
```

Comment:
@vanzod You'll need to use `# modextravars` to make the style check happy... (i.e., add a space after the `#`)